safegraph_py_functions/cbg_functions.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you add usage documentation, similar to lines 11-140 in `safegraph_py_functions.py`?

Reply:
Also need to add these functions (and import statement instructions) to the README.